airflow_valohai_plugin/operators/valohai_download_execution_outputs_operator.py

```diff
@@ -53,7 +53,6 @@ def get_output_path(self, name):
     def download_output(self, url, output_name):
         output_path = self.get_output_path(output_name)
         urlretrieve(url, output_path)
```

Comment:
While this wasn't changed in this PR, it seems this is considered as legacy interface. We may look into if it is desirable to use this method. Also, it has a `urlcleanup` method to clean temporary files, should we call it or can we expect temp files to get cleared by the system?

https://docs.python.org/3.7/library/urllib.request.html#urllib.request.urlretrieve